discord/ext/commands/converter.py

```diff
@@ -524,6 +525,33 @@ async def convert(self, ctx, argument):
         except Exception as exc:
             raise BadInviteArgument() from exc
 
+class GuildConverter(IDConverter):
+    """Converts to a :class:`~discord.Guild`.
+
+    The lookup strategy is as follows (in order):
```

Comment:
This should probably mention there's no disambiguation if guilds have multiple matching names.